process/slack_new_member_welcome.md

```diff
@@ -0,0 +1,42 @@
+<H1><img align="left" height=100 src="images/Girlgeek_w_workmark.png"><br> 
+  <em>Hello!</em></H1>
+
+&nbsp;  
+
+You're welcome to the Slack Platform of theGlobal PyLadies Community. 
+
+Kindly do well to introduce yourself in #introductions with the following information:
+
+
+
+- Your name
+- The PyLadies chapter you organize or belong to (_if you don't have a local one just say **"PyLadies"!**_)
+- Your pronouns (**e.g.** she/hers, they/them)
+- Tell us about your involvement with the Python Programming Language!
+
+
+
+While you can make posts anywhere on this platform, you are cautioned to put them in the right channel. We have many available channels, such as those listed below. However,  it may help you engage more effectively by finding the _appropriate_ channel to participate in! 
+
+Some great places to start: 
+
+\#introductions
+
+\#general
+
+\#conferences
+
+\#pyladies-support
+
+\#africa
```

Comment:
Just curious - why is specifically Africa listed? Maybe something more general like "check if your city has a channel by searching for your city name is slack channels like #nyc or #milan"

Reply:
Well noted @albendz.

Reply:
This was a channel created during `PyCon Africa` this year. If someone wants to go through and make all those channels, sure. But instead of adding more work in ... perhaps we can simply use language like:

*suggestion:*
`Check if your city or region has a channel by searching for your city name or your regional name in the Slack Channels! For example: #africa and #nyc`.

I think keeping this flexibility is good because some regions have much less PyLadies chapters and if you want to talk about regional issues, we do not want to dissuade people from starting/participating in channels outside of an immediate city.